Apps/APN-UIKit/APN UIKit/DIGraphShared.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
-public class DIGraph {
-    public static let shared: DIGraph = .init()
+public class DIGraphShared {
+    public static let shared: DIGraphShared = .init()
```

Comment:
Since our SDK template was updated to support only `DIGraphShared` earlier, and our sample app is reusing the same template, had to rename `DIGraph` to `DIGraphShared` for accurate auto-generated code. `make generate` wasn’t previously executed on sample apps, so I ran it on APN UIKit in this PR to ensure the changes are compilable.

Reply:
I am glad that the sample apps can re-use the template files so I am OK with this change.